inference-engine/src/mkldnn_plugin/nodes/mkldnn_def_conv_node.cpp

```diff
@@ -675,8 +671,9 @@ void MKLDNNDeformableConvolutionNode::initSupportedPrimitiveDescriptors() {
 
     impl_desc_type impl_type;
     const int simd_w = mayiuse(cpu::x64::avx512_common) ? 16 : 8;
-    if (group != 1 || (((getInputShapeAtPort(0).getStaticDims()[1] / group) % simd_w != 0)
-    || ((getOutputShapeAtPort(0).getStaticDims()[1] / group) % simd_w != 0))) {
+
+    if (group != 1 || (((getInputShapeAtPort(0).getDims()[1] / group) % simd_w != 0)
```

Comment:
you also need to check that that dims value != undefined

also in some situation you can get [? ? ? ?] data shape and [O I Y X] weight shape (in most cases weight is constant)
so  better to check weightShape[0] and weightShape[1] to choose jit version

Reply:
Done